### ISSUE TITLE
Invalid JSON file for stations from LDI

### ISSUE THREAD

**Issue by amotl** [panodata/luftdatenpumpe#17]
Users already reported that `luftdatenpumpe` would yield an invalid JSON file for the stations of luftdaten.info, see also https://source.irceline.be/corona-eu/luftdatenpumpe/issues/11.

Now, we have been able to confirm this on our reference dashboard, see https://weather.hiveeyes.org/grafana/d/AOerEQQmk/luftdaten-info-karte.

On the particular dashboard, it looks like:

![image](https://user-images.githubusercontent.com/453543/70431902-5277b600-1a7e-11ea-92f3-e8482cfce450.png)

At the particular location within the JSON file, it looks like that:
```
  {
    "key": "293",
    "name": "Mecklenburger Stra\u00dfe, Lenensruhe, Wismar, Nordwestmecklenburg, Mecklenburg-Vorpommern, DE"
  }
][
  {
    "key": "293",
    "name": "Mecklenburger Stra\u00dfe, Lenensruhe, Wismar, Nordwestmecklenburg, Mecklenburg-Vorpommern, DE"
  },
  {
    "key": "294",
    "name": "Reutlinger Stra\u00dfe, Neckarsulm, Baden-W\u00fcrttemberg, DE"
  },
```

**Comment by amotl** [comment]
We just found out that invoking Luftdatenpumpe like
```
luftdatenpumpe stations --network=ldi --source=${rdbms_uri} --target=json.grafana.kn+stream://sys.stdout
```
on our workstation did **not** yield duplicate stations within the invalid JSON output as outlined above.

**Comment by amotl** [comment]
The reason for this issue was that batch processing / flushing was also performed on JSON output. Oh my - don't do that.

**Comment by amotl** [comment]
Fixed by b7614986 within 0.19.1.
